src/main/java/seedu/address/model/export/DocumentFilePath.java

```diff
@@ -30,14 +30,14 @@
      */
     public DocumentFilePath(String documentFilePath) {
         requireNonNull(documentFilePath);
-        checkArgument(isValidDocumentFilePath(documentFilePath), MESSAGE_CONSTRAINTS);
+        checkArgument(isValid(documentFilePath), MESSAGE_CONSTRAINTS);
```

Comment:
Any reason why you changed to isValid? Since you are comparing a specific type of String (documentFilePath), a valid String for date might be different from a valid String for file path, might be good to keep it as isValidDocumentFilePath for clarity

Reply:
I suppose you're right.